extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -245,6 +250,7 @@ class AmpLightbox extends AMP.BaseElement {
   update_(pos) {
     dev().fine(TAG, 'update_');
     this.updateChildrenInViewport_(pos, this.oldPos_);
+    this.scrollHandler_();
```

Comment:
why here? see my [comment](https://github.com/ampproject/amphtml/pull/10009#discussion_r122829766) for where to call `this.scrollHandler`